extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js

```diff
@@ -101,21 +121,31 @@ export class QwcExtensions extends observeState(LitElement) {
                                 namespace="${extension.namespace}">
 
                              </${extension.card.componentName}>`;
-        
+
         return html`${unsafeHTML(customCardCode)}`;
-        
+
     }
 
     _renderDefaultCardContent(extension){
-        
-        return html`<div class="card-content" slot="content">
+        if(extension.status === "experimental") {
+            return html`
+                <div class="card-content" slot="content">
+                    <span class="description">
+                        <span class="float-right badge badge-warning">EXPERIMENTAL</span>
```

Comment:
You should use `qui-badge`. See https://github.com/quarkusio/quarkus/blob/main/extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/qui-badge.js

Here is an example where it's used:

https://github.com/quarkusio/quarkus/blob/b2f79310c2eb202a184838aa2fa4d558f03f9ecb/extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-server-log.js#L12

and 

https://github.com/quarkusio/quarkus/blob/b2f79310c2eb202a184838aa2fa4d558f03f9ecb/extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-server-log.js#L265

You would add `theme="warning"`

Thanks !